src/Microsoft.Extensions.Logging.Abstractions/Internal/NullLogger.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Logging.Abstractions.Internal
+{
+    public sealed class NullLogger : ILogger
```

Comment:
We shouldn't encourage APIs in the `.Internal` folder referenced by other projects in the same solution. This either needs to be proper public API or shared source